src/main/java/org/sonarlint/intellij/util/ConfigurationSharing.kt

```diff
@@ -93,6 +93,12 @@ class ConfigurationSharing {
                         )
                         SonarLintConsole.get(project).error("Error while creating the shared file, IO exception : " + e.message)
                     }
+
+                    get(project).simpleNotification(
```

Comment:
Move the notification after the the `BufferedWriter` wrote the file, otherwise it will get displayed even if it failed

Reply:
Nice catch